.circleci/config.yml

```diff
@@ -89,7 +89,6 @@ jobs:
           - v2-dependencies-{{ checksum "yarn.lock" }}
       # Download and cache dependencies
       - run: yarn
-      - run: yarn buildModules
```

Comment:
@alisman where else is it run? Can't seem to find it. Is it run as part of some other yarn job?

Reply:
@inodb `yarn build` runs `yarn buildModules` as of the latest setup

Reply:
gotcha - thanks!